app/src/main/java/org/oppia/android/app/parser/StringToNumberParser.kt

```diff
@@ -41,6 +41,9 @@ class StringToNumberParser {
    * detection should be done using [getRealTimeAnswerError], instead.
    */
   fun getSubmitTimeError(text: String): NumericInputParsingError {
+    if (text.normalizeWhitespace().isEmpty()) {
```

Comment:
Use `.isBlank()` to return if this string is empty or consists solely of whitespace characters.

Reply:
Done.